### ISSUE TITLE
Divider in app switcher

### ISSUE THREAD

**Issue by cedar-ave** [HealthCatalyst/Fabric.Cashmere#1393]
Enable the app switcher to be divided into sections. As an example, see "Help" with the divider line beneath in the screenshot. The term used would be customizable. (In this example screenshot, it is intended as a placeholder term only.)

![app-switcher](https://user-images.githubusercontent.com/15255009/94043498-426d6080-fd8a-11ea-91f1-8fc018b00c7e.png)

**Comment by joeskeen** [comment]
Would this require the discovery service to be changed to include a category or tag field?

**Comment by andrew-frueh** [comment]
Yes @joeskeen I think we'd need a category or tag on items in the discovery service so we know how to categorize them.

**Comment by andrew-frueh** [comment]
`service_type` is a field that can be used as a way to identify the "group" from the Discovery Service
